flowsa/settings.py

```diff
@@ -84,8 +83,23 @@
 vLog.addHandler(vLog_fh)
 vLogDetailed.addHandler(vLog_fh)
 
+
+def return_pkg_version():
```

Comment:
If we find this works well we could consider moving to esupy similar to `get_git_hash()`

Reply:
@bl-young Yes, agreed. I do think it is a better method than manually updating a parameter in a .py file